Advanced_Projects/Resume Scanner/extractor.py

```
@@ -0,0 +1,9 @@
+
+from PyPDF2 import PdfReader
+
+def extract_text_from_pdf(file_path):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)